src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -205,7 +202,29 @@ public JSONMessage color(ChatColor color) {
         last().setColor(color);
         return this;
     }
-
+    
+    /**
+     * Sets the color of the current message part.
+     * <br>This method allows you to provide a hex color, which can be used in 1.16 to color the message in any Hex color
+     * imaginable.
+     * 
+     * @param color The color to set
+     * @return This {@link JSONMessage} instance
+     */
+    public JSONMessage color(String color) {
+        if(color.startsWith("#") && ReflectionHelper.MAJOR_VER < 16) {
+            throw new IllegalArgumentException("Hex colors are only supported on 1.16+!");
```

Comment:
hmmmmm. we obviously want some kind of feedback if this is being used on an unsupported version to make debugging easier, but without any kind of feature flag system, handling that kind of version gating at the consumer level is somewhat cumbersome. perhaps two methods should be provided:

```
color(String hexColor, ChatColor fallbackColor)
color(String hexColor)
```
where the first would use the fallback if on older versions, and the second would either do a one-time log warning that it's not supported, or simply noop on older versions?

thoughts?

Reply:
I personally think to just have the two methods and default to color x when hex isn't valid.
We could perhaps require a JavaPlugin or Logger instance, that people would provide when getting the JSONMessage instance, to then have getLogger.warn, but I don't see how this would be useful, if you use this in several places.

Reply:
Implemented a `color(String, ChatColor)` and made `color(String)` default to `WHITE` when the provided color is hex on a not 1.16 server

Reply:
I mean we can always create a Logger instance for the class, which can be used for logging a warning, but yeah that sounds good to just have the two methods like that. sounds good, will re-review this :)